kakarot_scripts/constants.py

```diff
@@ -19,8 +19,9 @@
 logger.setLevel(logging.INFO)
 load_dotenv()
 
-# Hardcode block gas limit to 7M
+# Hardcode block gas limit to 7M if this is changed, also update the MAX_MEMORY_SIZE in the Kakarot contract
```

Comment:
give more context: max memory size to have memory expansion cost fitting in block gas limit

Reply:
Also add that this memory size actually gives `7_000_130` gas, so it's just a little bit more and consequently is safe to use a flag for out of gas